templates/ingress.yaml

```diff
@@ -16,28 +17,8 @@ metadata:
 {{ toYaml . | indent 4 }}
 {{- end }}
   annotations:
-    kubernetes.io/ingress.class: {{ .Values.ingress.class }}
-    kubernetes.io/tls-acme: {{ eq .Values.ingress.acme true | quote }}
-    nginx.ingress.kubernetes.io/custom-http-errors: "403,404,502,503"
-  {{- if .Values.ingress.auth.enabled }}
-    nginx.ingress.kubernetes.io/auth-url: https://houston.{{ .Values.ingress.baseDomain }}/v1/authorization
-    nginx.ingress.kubernetes.io/auth-signin: https://app.{{ .Values.ingress.baseDomain }}/login
-    nginx.ingress.kubernetes.io/auth-response-headers: authorization, username, email
-    nginx.ingress.kubernetes.io/proxy-cookie-path: / /{{ .Release.Name }}/
-    # To support people who are using Airflow's experimental API on the old URL, but don't
-    # have their client set up to follow redirects correctly: We proxy it to the right location
-    # for them.
-    nginx.ingress.kubernetes.io/server-snippet: |
-      location ^~ /api/ {
-        if ($host = '{{- include "deployments_subdomain" . }}' ) {
-          return 404;
-        }
-        proxy_pass {{ printf "https://%s/%s/airflow/$request_uri" ( include "deployments_subdomain" .) .Release.Name }};
-      }
-    nginx.ingress.kubernetes.io/configuration-snippet: |
-      if ($host = '{{- include "airflow_subdomain" . }}' ) {
-        return 308 {{ printf "https://%s/%s/airflow/$request_uri" ( include "deployments_subdomain" .) .Release.Name }};
-      }
+  {{- range $k, $v := .Values.ingress.webserverAnnotations }}
```

Comment:
how does this template out when webserverAnnotations is empty dict {} or null ~ ?

Reply:
@sjmiller609 thank you for catching that. Go templates FTL :(

Reply:
np, we really need something for unit testing helm charts for this exact kind of thing. https://github.com/astronomer/issues/issues/909